themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -4,24 +4,44 @@ doLogOutAllSessions=Desconectar de todas las sesiones
 doRemove=Eliminar
 doAdd=A\u00F1adir
 doSignOut=Desconectar
+doLogIn=Iniciar sesi\u00F3n
+doLink=Link
+
 
 editAccountHtmlTitle=Editar cuenta
+personalInfoHtmlTitle=Informaci\u00F3n personal
 federatedIdentitiesHtmlTitle=Identidades federadas
 accountLogHtmlTitle=Registro de la cuenta
 changePasswordHtmlTitle=Cambiar contrase\u00F1a
+deviceActivityHtmlTitle=Actividad del dispositivo
 sessionsHtmlTitle=Sesiones
 accountManagementTitle=Gesti\u00F3n de Cuenta Keycloak
 authenticatorTitle=Autenticador
 applicationsHtmlTitle=Aplicaciones
+linkedAccountsHtmlTitle=Cuentas vinculadas
+
+accountManagementWelcomeMessage=Bienvenido a la administraci\u00F3n de cuentas de keycloak
+personalInfoIntroMessage=Administre su informaci\u00F3n b\u00E1sica
+accountSecurityTitle=Seguridad de la cuenta
+accountSecurityIntroMessage=Controla tu contrase\u00F1a y acceso a la cuenta
+applicationsIntroMessage=Rastree y administre el permiso de su aplicaci\u00F3n para acceder a su cuenta
+resourceIntroMessage=Comparta sus recursos entre los miembros del equipo.
+passwordLastUpdateMessage=Su contrase\u00F1a fue actualizada en
+updatePasswordTitle=Actualizar contrase\u00F1a
+updatePasswordMessageTitle=Aseg\u00FArese de elegir una contrase\u00F1a segura
+updatePasswordMessage=Una contrase\u00F1a segura contiene una combinaci\u00F3n de n\u00FAmeros, letras y s\u00EDmbolos. Es dif\u00EDcil de adivinar, no se parece a una palabra real y solo se usa para esta cuenta.
+personalSubTitle=Informaci\u00F3n personal
+personalSubMessage=Administre esta informaci\u00F3n b\u00E1sica: su nombre, apellido y correo electr\u00F3nico
```

Comment:
Better to use the translation "Gestionar" of "Manage" like in other translations in the same file.
Also "apellidos" in plural.